projects/dsp-ui/src/lib/action/services/notification.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@angular/core';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { ApiResponseError } from '@dasch-swiss/dsp-js';
+import { StatusMsg } from '../../../assets/i18n/statusMsg';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class NotificationService {
+
+    constructor(
+        private _snackBar: MatSnackBar,
+        private _statusMsg: StatusMsg
+    ) { }
+
+
+    // TODO: maybe we can add more parameters like:
```

Comment:
Why actually duration couldn't be a const?

Reply:
We can set it as a const. The question is: how long should we display a notification? 15 seconds can be to long for success messages but it's okay for the server error messages. We can use a formula as suggest in the second answer here: <https://ux.stackexchange.com/questions/11203/how-long-should-a-temporary-notification-toast-appear> 😄
@mpro7 Would you like to update this PR with best practices? Thanks

Reply:
According to the link above and our longest possible message (correct me if I'm wrong) "The request was a legal request, but the server is refusing to respond to it. For use when authentication is possible but has failed or not yet been provided" I suggest to set it to 5 seconds.

Reply:
Youtrack also displays a snackbar notification in case of server error; the duration there is more than 10 seconds. When we have a server error (db is not running (500) or the whole stack (503)) nothing will work anymore. In this case the notification should be there for longer time and maybe should include a link to `refresh` the page e.g. "Come back in a few minutes and try to <relaod the page>". Already discussed in DSP-710

Reply:
As agreed during call the duration will remain as const set to 5 seconds until next possible iteration, when the behaviour of the service will be enriched.